api/graphql/schema.graphql

```diff
@@ -796,10 +796,17 @@ enum CandidateExpiryFilter {
   ALL
 }
 
+enum CandidateSuspendedFilter {
+  ACTIVE
+  SUSPENDED
+  ALL
+}
+
 input UserPoolFilterInput {
   poolId: ID!
   expiryStatus: CandidateExpiryFilter = ACTIVE
   statuses: [PoolCandidateStatus!]
+  suspendedStatus: CandidateSuspendedFilter = ALL
```

Comment:
I was a bit iffy on this bit, wasn't sure if it was needed to match this up with the possible fields in `scopePoolFilters`